src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -231,44 +231,6 @@ asset::SAssetBundle COBJMeshFileLoader::loadAsset(io::IReadFile* _file, const as
                     else
                     {
                         submeshes.push_back(core::make_smart_refctd_ptr<ICPUMeshBuffer>());
-
```

Comment:
what's going on here?

Reply:
just deleted some commented-out code i should have deleted a while ago